src/StoreApi/Schemas/V1/AbstractAddressSchema.php

```diff
@@ -95,64 +99,12 @@ public function sanitize_callback( $address, $request, $param ) {
 		$address['address_1']  = wc_clean( wp_unslash( $address['address_1'] ) );
 		$address['address_2']  = wc_clean( wp_unslash( $address['address_2'] ) );
 		$address['city']       = wc_clean( wp_unslash( $address['city'] ) );
-		$address['state']      = $this->format_state( wc_clean( wp_unslash( $address['state'] ) ), $address['country'] );
+		$address['state']      = $validation_util->format_state( wc_clean( wp_unslash( $address['state'] ) ), $address['country'] );
 		$address['postcode']   = $address['postcode'] ? wc_format_postcode( wc_clean( wp_unslash( $address['postcode'] ) ), $address['country'] ) : '';
 		$address['phone']      = wc_clean( wp_unslash( $address['phone'] ) );
 		return $address;
 	}
 
-	/**
-	 * Get list of states for a country.
-	 *
-	 * @param string $country Country code.
-	 * @return array Array of state names indexed by state keys.
-	 */
-	protected function get_states_for_country( $country ) {
```

Comment:
This is used in `AbstractAddressSchema`. Removing it will break that implementation. TBF this method doens't really do much so feel free to just move the code inline to remove the need for the util in that class.